en/docs/develop/restapis/applications.yaml

```diff
@@ -0,0 +1,2962 @@
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/OpenIDConnectConfiguration'
+        description: >-
+          This represents the OIDC authentication protocol parameters of an
+          application.
+        required: true
+    delete:
+      tags:
+        - Inbound Protocols - OAuth / OIDC
+      summary: |
+        Delete OIDC authentication protocol parameters of an application.
+      description: >
+        This API provides the capability to delete OIDC authentication protocol
+        parameters of an application.
+      operationId: deleteInboundOAuthConfiguration
+      parameters:
+        - name: applicationId
+          in: path
+          description: Id of the application.
+          required: true
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Delete Success
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+
+  '/applications/{applicationId}/inbound-protocols/oidc/regenerate-secret':
+    post:
+      tags:
+        - Inbound Protocols - OAuth / OIDC
+      summary: |
+        Regenerate the OAuth2/OIDC client secret.
+      description: |
+        This API provides regenerate the OAuth2/OIDC client secret.
```

Comment:
This API provides **the capability to** regenerate the OAuth2/OIDC client secret.

or

This API regenerates the OAuth2/OIDC client secret.